src/core/shape/attributes.js

```diff
@@ -15,7 +15,7 @@ import * as constants from '../constants';
  * By default, the first two parameters of
  * <a href="#/p5/ellipse">ellipse()</a>, <a href="#/p5/circle">circle()</a>,
  * and <a href="#/p5/arc">arc()</a>
- * are the x- and y-coordinates of the shape's center. The next parameters set
+ * are the x- and y-coordinates of the shape's upper left corner. The next parameters set
```

Comment:
@nickmcintyre do you know if this one needed correcting? Looks like it's the center in the current docs https://p5js.org/reference/#/p5/ellipse so `ellipse` might be good as is

Reply:
As per my tests, the x and y coordinates of ellipse() determine the position of the center of the figure by default, so its description is accurate.

Reply:
Sorry I see the problem now. This is the first time I create a pull request and seems like I created commit 4892c6f by mistake. I only meant to commit cda061b

Is there a way I can delete 4892c6f so only the right one gets pulled?

Reply:
No problem! If you run `git revert 4892c6f` and push the results, then the Files Changed tab on the PR will only show the changes from cda061b.

Reply:
Can I run that command somewhere from GitHub? I used Codespaces to make those commits. Can you please send me some quick steps on how to do it? Thanks for the help

Reply:
Ah looks like you got it!

Reply:
Yeah I think so lol. Is there something else I should do?

Reply:
I think it should be good, I'm just waiting for tests to pass and then I'll merge if they look good! (and they should since its just a change to the reference text.)

Reply:
Great! It is a very simple contribution, but because it is the first in some way it is special for me haha. Thanks!

Reply:
Thanks, we appreciate it!